manager/state/raft/transport/mock_raft_test.go

```diff
@@ -58,9 +58,10 @@ func newMockRaft() (*mockRaft, error) {
 		updatedNodes:         make(chan updateInfo, 4096),
 	}
 	cfg := &Config{
-		HeartbeatInterval: 3 * time.Second,
-		SendTimeout:       2 * time.Second,
-		Raft:              mr,
+		HeartbeatInterval:   3 * time.Second,
+		SendTimeout:         2 * time.Second,
+		SendTimeoutSnapshot: 20 * time.Second,
```

Comment:
I'd still suggest making this (and others too) a constant and using it here and in raft.go

Reply:
constants won't make this clearer, just harder to read.

Reply:
Well, its better to have it defined in one place than inline in many places.

Reply:
Let's do that in a follow-up if we think it's required.

Reply:
Isn't that a simple change ?